src/org/opensolaris/opengrok/configuration/messages/NormalMessage.java

```diff
@@ -18,37 +18,38 @@
  */
 
  /*
- * Copyright (c) 2016, 2017, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2016, 2018, Oracle and/or its affiliates. All rights reserved.
  */
 package org.opensolaris.opengrok.configuration.messages;
 
-import org.opensolaris.opengrok.configuration.RuntimeEnvironment;
+import java.util.Collections;
+import java.util.Set;
 
 /**
  * @author Kryštof Tulinger
  */
 public class NormalMessage extends Message {
```

Comment:
This will break `Messages` usage. No problem with that? Now there will have to be `-n info` option. Maybe because it is default there won't be problem with that?

Reply:
I'm for this definitely.